external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutEmitter.java

```diff
@@ -279,6 +292,18 @@ private long seek(TopicPartition tp, KafkaTridentSpoutBatchMetadata lastBatchMet
         return fetchOffset;
     }
 
+    /**
+     * Seek the consumer to offset corresponding to startTimeStamp.
+     */
+    private void seekOffsetByStartTimeStamp(TopicPartition tp) {
+        long startTimeStampOffset;
```

Comment:
Nit: No reason to not declare and assign on the same line.

Reply:
yes. moved